DNSecure/Models/Presets.swift

```diff
@@ -44,8 +44,8 @@ enum Presets {
                     servers: [
                         "1.1.1.1",
                         "1.0.0.1",
-                        "2606:4700:4700::64",
-                        "2606:4700:4700::6400",
+                        "2606:4700:4700::1111",
+                        "2606:4700:4700::1001",
```

Comment:
Thanks for fixing these addresses. It seemed that I had been using addressed for DNS64.

https://developers.cloudflare.com/1.1.1.1/infrastructure/ipv6-networks/